client/landing/gutenboarding/gutenboard.tsx

```diff
@@ -56,25 +50,30 @@ export function Gutenboard() {
 	const [ isEditorSidebarOpened, updateIsEditorSidebarOpened ] = useState( false );
 	const toggleGeneralSidebar = () => updateIsEditorSidebarOpened( isOpen => ! isOpen );
 
-	const onboardingState = useSelect( select => select( STORE_KEY ).getState() );
+	const { siteVertical } = useSelect( select => select( STORE_KEY ).getState() );
 
-	// FIXME: Quick'n'dirty step state, replace with router
-	const [ currentStep, setStep ] = useState( Steps.IntentGathering );
-	const goToNextStep = stepCompleted[ currentStep ]( onboardingState )
-		? () => setStep( step => step + 1 )
-		: undefined;
-	const goToPrevStep = currentStep > 0 ? () => setStep( step => step - 1 ) : undefined;
+	// @TODO: This is currently needed in addition to the routing (inside the Onboarding Block)
+	// for the 'Back' and 'Next' buttons in the header. If we remove those (and move navigation
+	// entirely into the block), we'll be able to remove this code.
```

Comment:
For this router to be compelling I'd say this needs to go.

Reply:
As discussed on a call, this is currently breaking encapsulation of routing, which for the most part lives inside of the onboarding block -- replacing a previous `switch`/`case` statement with a `<Switch />`/`<Route />` construct quite literally, mostly for continuity's sake.

Keeping a bit of routing logic here for now seems okay, since we expect the Back and Next button to move into the block, thus allowing encapsulation of routing logic there. Otherwise, we can always move the `<Switch />`/`<Route />` construct up a level (into `gutenboard.tsx`). We should keep separation of concerns in mind tho, especially as we implement the customized editor.